compiler+runtime/src/cpp/jank/runtime/obj/atom.cpp

```diff
@@ -111,7 +111,9 @@ namespace jank::runtime::obj
     while(true)
     {
       auto v(val.load());
-      auto const next(apply_to(fn, conj(a1, conj(a2, rest))));
+      auto const args(
+        jank::runtime::cons(v, jank::runtime::cons(a1, jank::runtime::cons(a2, rest))));
```

Comment:
No need for `jank::runtime::` in any of these lines. We're already in the `jank::runtime::obj` namespace.

Reply:
Ah, in that case, we use `runtime::cons`. Basically, use as little qualification as possible. You should basically never need `jank::` within the jank code.